src/App.vue

```diff
@@ -522,46 +522,15 @@ export default {
       this.setGoodStanding(business.goodStanding)
       this.setHasCourtOrders(business.hasCourtOrders)
 
-      if (business.state === EntityState.HISTORICAL) {
-        await this.parseStateFiling(business.stateFiling) // will throw on error
+      // retrieve stateFiling if the URL is included in the business payload
+      if (business.stateFiling) {
+        await this.retrieveStateFiling(business.stateFiling)
```

Comment:
I see errors are caught by the grand-parent of the current method. Is that the plan? Is it tested?

Reply:
You are correct, the retrieveStateFiling() method sits inside a try / catch block.  If the method is unsuccessful, it throws an error and parent executes the catch block code.  I've been careful not to change the behavior.